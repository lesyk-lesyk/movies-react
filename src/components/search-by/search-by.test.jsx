import React from 'react';
import { SearchBy } from './index';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

jest.mock('../search-mode-button/search-mode-button');

describe('SearchBy', () => {
  it('renders', () => {
    const tree = renderer
      .create(<SearchBy />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
