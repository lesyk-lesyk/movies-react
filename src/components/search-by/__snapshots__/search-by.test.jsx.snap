// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBy renders 1`] = `
<div
  className="search-by"
>
  <span
    className="caption"
  >
    Search by
  </span>
  <SearchModeButton
    mode="title"
  >
    Title
  </SearchModeButton>
  <SearchModeButton
    mode="genres"
  >
    Genre
  </SearchModeButton>
</div>
`;
